Modules/Core/Common/test/itkPeriodicBoundaryConditionTest.cxx

```diff
@@ -308,9 +308,9 @@ itkPeriodicBoundaryConditionTest(int, char *[])
     for (auto & j : back)
     {
       const float d = testIter.GetPixel(j);
-      sum += d * d;
```

Comment:
I guess this was here to make sure compiler does not optimize away the call on the previous line.

Reply:
OK, I now use this dummy sum and log it.